### PR TITLE
Sdk 655 add cols

### PR DESCRIPTION
**Pull request by b-pmcg**

### CODE REVIEW

File: src/references/ilkList.js [hunk 1]
```diff
@@ -44,11 +44,11 @@ export default [
...
-  //   gem: 'COL5',
-  //   currency: COL5
-  // }
+  {
+    slug: 'col5-a',
+    symbol: 'COL5-A',
+    key: 'COL5-A',
+    gem: 'COL5',
+    currency: COL5
+  }
```
Comment (abylaw) [inline src/references/ilkList.js]: Are we not adding COL2-4? 🤔

Comment (b-pmcg) [reply src/references/ilkList.js]: They weren't in the testnet.json, which was causing problems with the mcd plugin, but with @ethanbennett's PR here, they will be ready to be added in https://github.com/makerdao/dai.js/pull/127/files#diff-9d66bae500be515575fa289f1d9ebcd4R4

File: src/components/CDPCreateScreens/CDPCreateConfirmCDP.js [hunk 2]
```diff
@@ -184,13 +184,16 @@ const CDPCreateConfirmCDP = ({ dispatch, cdpParams, selectedIlk, onClose }) => {
     const { type } = payload;
     if (type !== 'increment-step') return dispatch(payload);
 
+    const decimals = maker.service('token').getToken(selectedIlk.data.gem)
```
Comment (jparklev) [inline src/components/CDPCreateScreens/CDPCreateConfirmCDP.js]:
```suggestion
    const decimals = maker.getToken(selectedIlk.currency)
```

File: src/references/ilkList.js (hunk 1 above)
Comment (b-pmcg) [reply src/references/ilkList.js]: @abylaw I just published a new version of the MCD plugin which allows us to use the rest of these COLs, so COL1-5 are available to use now :-)

**Pull request merged**